content/docs/typechecking-with-proptypes.md

```diff
@@ -157,9 +161,7 @@ class Greeting extends React.Component {
 }
 
 // Specifies the default values for props:
-Greeting.defaultProps = {
-  name: 'Stranger'
-};
+Greeting.defaultProps = defaultProps;
```

Comment:
I don't understand the motivation behind this change. Compatibility with AirBnb's style-guide seems a bit arbitrary and not necessarily something we need to aim for within the examples shown here.

[AirBnb's own documentation](https://github.com/airbnb/javascript/tree/master/react) shows both formats:
```js
const propTypes = {
  id: PropTypes.number.isRequired,
  url: PropTypes.string.isRequired,
  text: PropTypes.string,
};

const defaultProps = {
  text: 'Hello World',
};

class Link extends React.Component {
  static methodsAreOk() {
    return true;
  }

  render() {
    return <a href={this.props.url} data-id={this.props.id}>{this.props.text}</a>;
  }
}

Link.propTypes = propTypes;
Link.defaultProps = defaultProps;
```
```js
// good
function SFC({ foo, bar, children }) {
  return <div>{foo}{bar}{children}</div>;
}
SFC.propTypes = {
  foo: PropTypes.number.isRequired,
  bar: PropTypes.string,
  children: PropTypes.node,
};
SFC.defaultProps = {
  bar: '',
  children: null,
};
```

This change would also be internally inconsistent with some of the examples above which declare `propTypes` inline. (Why would we treat `defaultProps` differently?)

I tend to think we should revert this bit. 😄

Reply:
Thank you for your reply, and I will update this commit.